### ISSUE TITLE
-> One of the tabs of website- What's going on in college

### ISSUE THREAD

**Issue by ChetnaNSharma** [ChetnaNSharma/Alumni-Connect#22]
![20210328_162444](https://user-images.githubusercontent.com/80457581/112749842-ca226a80-8f79-11eb-90b3-3696b3d1c2a7.jpg)
I need some really unique and elegant designs for the four pages linked to this tab.

**Comment by ChetnaNSharma** [comment]
Call for participants who are willing to build a real creative page!

**Comment by riasenpai** [comment]
I would like to work on this and add the feat mentioned. @ChetnaNSharma . How much time will I get to complete the task and what level is this issue of?

**Comment by ChetnaNSharma** [comment]
Hi Ria
I am thankful to your response.
This tab has to be done completely within 25 days. Did you check the website I mentioned? Would you be able to bring those effects?

**Comment by riasenpai** [comment]
> Hi Ria
> I am thankful to your response.
> This tab has to be done completely within 25 days. Did you check the website I mentioned? Would you be able to bring those effects?

@ChetnaNSharma 
Yes I checked, its the zoom in effect while scrolling down. Could you also mention the tech stack to be used here and label this issue with Gssoc tag and level tag too so that I can start contributing ASAP. I would be updating you with the progress over the discord channel, please assign this to me. Thank you

**Comment by riasenpai** [comment]
@ChetnaNSharma  @yash982000 There has been a PR for login page by my fellow contributor. So please merge it because this project has no boilerplate as of yet. I could then pull the login page and initialize a boiler plate in ReactJS because if everyone has a different theme and no proper initial bootstrapping and direction for this project, then there will be merge conflicts which would be difficult to solve.

**Comment by yash982000** [comment]
> @ChetnaNSharma @yash982000 There has been a PR for login page by my fellow contributor. So please merge it because this project has no boilerplate as of yet. I could then pull the login page and initialize a boiler plate in ReactJS because if everyone has a different theme and no proper initial bootstrapping and direction for this project, then there will be merge conflicts which would be difficult to solve.

@riasenpai We understand but this issue was earlier assigned to someone. Now, there has been many changes in the repo and it will create a merge conflict. You may open a fresh issue for whatever new changes you want to contribute. 

Also, please do list tag the PR to which you are referring. There are a number of issues and PR for us to handle which makes it difficult to track the participants if the PR is not referneced.

**Comment by yash982000** [comment]
@riasenpai Are you still on here?

**Comment by yash982000** [comment]
> @riasenpai Are you still on here?

Any updates? @riasenpai  

Let me know on WhatsApp or discord so that we can close this issue or assign it appropriately.
DM on discord if you need my contact info for discussing your work or any other doubts.

**Comment by Mukta-Sawant** [comment]
@yash982000 @ChetnaNSharma I would like to work on this as a part of GSSOC'21. Please assign it to me.

**Comment by SnehaVeerakumar** [comment]
@yash982000 was this issue solved ?

**Comment by yash982000** [comment]
@Mukta-Sawant @SnehaVeerakumar 
This issue is mostly not done yet. You can check and verify the issues and PR and let me know if it is yet to be done.
I will assign any of you who comes up first as we have a rule of First Come, First Serve.

**Comment by Mukta-Sawant** [comment]
@yash982000 yes it's not done yet.

**Comment by yash982000** [comment]
You may proceed then.

**Comment by SnehaVeerakumar** [comment]
> You may proceed then.

Me or @Mukta-Sawant ?

**Comment by yash982000** [comment]
> > You may proceed then.
> 
> Me or @Mukta-Sawant ?

I may not decide. You both can mutually decide and take it up because both of you have been around this issue at the same time.

**Comment by SnehaVeerakumar** [comment]
> > > You may proceed then.
> > 
> > 
> > Me or @Mukta-Sawant ?
> 
> I may not decide. You both can mutually decide and take it up because both of you have been around this issue at the same time.

I don't want to create any kinda disputes, I will take up some other issue :)

Go ahead @Mukta-Sawant

**Comment by Mukta-Sawant** [comment]
@SnehaVeerakumar We can work on this together, @yash982000 will that be okay with you?

**Comment by yash982000** [comment]
> @SnehaVeerakumar We can work on this together, @yash982000 will that be okay with you?

@Mukta-Sawant, @SnehaVeerakumar I do not have any problem. But, will need to divide the scores mostly. Maybe, if you have some suggestion on that, I am open to hear.

**Comment by SnehaVeerakumar** [comment]
The other guy can work on this issue as I will not be available for few days :)

**Comment by yash982000** [comment]
> The other guy can work on this issue as I will not be available for few days :)

@SnehaVeerakumar OKay, thanks for informing fast. 
@Mukta-Sawant The stage is all yours now.

**Comment by Mukta-Sawant** [comment]
@yash982000 yes thank you.

**Comment by SnehaVeerakumar** [comment]
> > The other guy can work on this issue as I will not be available for few days :)
> 
> @SnehaVeerakumar OKay, thanks for informing fast.
> @Mukta-Sawant The stage is all yours now.

Before 7 days I was absolutely free and I was waiting for your reply. Now that I have registered for MLH Hackathon, I replied I will not be available. As your statement sounded ironical, I gave my explanation.

**Comment by yash982000** [comment]
> > > The other guy can work on this issue as I will not be available for few days :)
> > 
> > 
> > @SnehaVeerakumar OKay, thanks for informing fast.
> > @Mukta-Sawant The stage is all yours now.
> 
> Before 7 days I was absolutely free and I was waiting for your reply. Now that I have registered for MLH Hackathon, I replied I will not be available. As your statement sounded ironic, I gave my explanation.

@SnehaVeerakumar What sounded you ironical?
I genuinely thanked you for replying fast.

And, sorry for not able to reply earlier due to some commitments. Also, all the best for your hackathon. You still have time to come back with a new feature or issue. :)

**Comment by varmor** [comment]
Assign me this issue.
I would like to work on it.

**Comment by yash982000** [comment]
> Assign me this issue.
> I would like to work on it.

I think @Mukta-Sawant is working on this. 
If not , I will assign you in a couple of days. @VarunMore

**Comment by Mukta-Sawant** [comment]
@yash982000 Sir I had some doubts about the issue, so to whom exactly I've to ask? I tried to talk with @ChetnaNSharma but due to some reasons, she's not active on discord.

**Comment by yash982000** [comment]
@Mukta-Sawant Can you please DM me on whatsapp at +91 9106584241?
I will share her contact details.
